dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/TrackedEntityAttributeController.java

```diff
@@ -223,4 +225,29 @@ private Map<String, String> getRequiredValues( TrackedEntityAttribute attr, Map<
         return result;
     }
 
+    // --------------------------------------------------------------------------
+    // Hooks
```

Comment:
Not sure if we have a "standard"/"guideline" for writing such comments. IMHO these do not add anything useful, other than us having to scroll more 😋

Reply:
Yea agreed. It was part of the copy paste from another Controller class that implemented the hooks. Removed it here.